api/routes/addEntries.js

```diff
@@ -1,37 +1,27 @@
+//Add New entries
 const express = require('express')
 const route = express.Router()
 const database = require('../models/post.model')
-
 route.post('/addEntries', (req,res)=>{
-    const {title, description} = req.body
-    const databaseLength = database.Posts.length
-    let oldId = database.Posts[databaseLength - 1].id
-    let newId = oldId 
-    database.Posts.push({
-        id:newId + 1,
-        title:title,
-        description:description,
-        createdOn: Date()
-    })
+const {title, description} = req.body
+const databaseLength = database.Posts.length
+let oldId = database.Posts[databaseLength - 1].id
+let newId = oldId 
+database.Posts.push({
+id:newId + 1,
+title:title,
+description:description,
+createdOn: Date()
+})
 res.json({
```

Comment:
I suggest you send status code with your response, eg 
`res.status(200).json({
....
})`

You can read more on status code here